lib/esa_feeder/gateways/esa_client.rb

```diff
@@ -20,7 +20,7 @@ def find_templates(tag)
       def create_from_template(post, user)
         response = driver.create_post(template_post_id: post.number, user: user)
         # error happen when user does not exists
-        raise PostCreateError, response.to_s if response.body['error']
+        raise PostCreateError, response.body if response.body['error']
```

Comment:
例外の文言をより読みやすくするために`response.body['message']` だけを渡すのはどうでしょうか？`error` と `message` の内容が重複しているので、`message` だけでいいかなぁと思いました。

```suggestion
        raise PostCreateError, response.body['message'] if response.body['error']
```

Reply:
はい、良さそうです！ 👍

Reply:
動作確認をし直して PR description も更新しました。